Atlas/backend/atlas/tests/test_comment.py

```diff
@@ -143,3 +143,54 @@ def test_create_cultural_heritage_item_with_multiple_comments(self):
         self.assertEqual(response.status_code, 200)
         response_content = json.loads(smart_text(response.content))
         self.assertEqual(len(response_content['comments']), 2)
+
+
+    def test_create_cultural_heritage_item_with_comment_and_user(self):
+        item_data = {
+            "title": "Very emotional thresh hook",
+        }
+        response = self.client.post(
+            self.cultural_heritage_item_url,
+            item_data,
+            format='json',
+
+        )
+        response_content = json.loads(smart_text(response.content))
+        id = response_content['id']
+        self.assertEqual(response.status_code, 201)
+        text = 'That is a nice heritage item'
+        item_data = {
+            'comment':
+                {'text': text,}
+
+        }
+        response = self.client.post(
+            self.cultural_heritage_item_url + str(id) + '/comment',
+            item_data,
+            format='json',
+
+        )
+        self.assertEqual(response.status_code, 201)
+        text = 'That is a nice heritage item2'
+        item_data = {
+            'comment':
+                {'text': text, }
+
+        }
+        response = self.client.post(
+            self.cultural_heritage_item_url + str(id) + '/comment',
+            item_data,
+            format='json',
+
+        )
+        self.assertEqual(response.status_code, 201)
+        response = self.client.get(
+            self.cultural_heritage_item_url + str(id) + '/comment',
```

Comment:
I think we need to make sure that when we send a request to get a cultural heritage item its comments will have the necessary information. Right now you are sending it to comment. Comment endpoint shouldn't have get option. Can you change this test as getting cultural heritage item instead of the comment itself?

Reply:
I think we should also get comments from the endpoint because it provides pagination while getting comments. Say if we have 1000 comments under an item @SaitTalhaNisanci

Reply:
I updated it to be list and create view

Reply:
@SaitTalhaNisanci  The comment creation already returns that info

Reply:
Well that endpoint will not return 1000 comments. You will render that page after making a HTTP GET request to get that cultural heritage item.